presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +However in many cases increasing this value will       +
++                                          +                                          +                                    +visibly increase computation time while writing.       +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++        ``query.execution-policy``        +``String`` (``all-at-once`` or ``phased``)+          ``all-at-once``           +Serves as default value for ``execution_policy``       +
++                                          +                                          +                                    +session property. Setting this value to ``phased``     +
++                                          +                                          +                                    +will allow query scheduler to split a single query     +
++                                          +                                          +                                    +execution between different time slots. This will      +
++                                          +                                          +                                    +allow to switch context more often and possibly stage  +
++                                          +                                          +                                    +the partially executed query in order to increase      +
++                                          +                                          +                                    +robustness. Average time of executing query may        +
++                                          +                                          +                                    +slightly increase after setting this to ``phased`` due +
++                                          +                                          +                                    +to context switching and more complex scheduling       +
++                                          +                                          +                                    +algorithm but drop in variation of query execution     +
++                                          +                                          +                                    +time is expected.                                      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++    ``query.initial-hash-partitions``     +               ``Integer``                +               ``8``                +Serves as default value for ``hash_partition_count``   +
```

Comment:
Isn't this config option somehow related to the number of nodes in the cluster? If so, why the default is 8? Should say this needs to be adjusted based on the cluster size?